src/app.js

```diff
@@ -1 +1,39 @@
+/* eslint-disable no-console */
 'use strict';
+
+const rl = require('readline');
+const { generateNumber } = require('./generateNumber');
+const { calculating } = require('./calculating');
+
+const number = generateNumber();
+
+const terminal = rl.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function playGame(startMessage) {
+  terminal.question(startMessage, (answer) => {
+    const { bulls, cows } = calculating(number, answer);
+
```

Comment:
The check for the length of the answer should be before the call to `calculating`. This would prevent unnecessary calculations for invalid inputs.